src/atopile/_shim.py

```diff
@@ -47,7 +41,7 @@ def _is_int(name: str) -> bool:
     return True
 
 
-class _has_kicad_footprint_name_defined(F.has_footprint_impl):
+class has_local_kicad_footprint_named_defined(F.has_footprint_impl):
```

Comment:
Hopefully we can replace this from this shim momentarily w/ your new explicit part pick